src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab | undefined) => void,
+};
+
+export const Tabs:React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const handleClick = (target: EventTarget & HTMLAnchorElement) => {
+    const currTabId = target.getAttribute('href')?.slice(1);
+
+    if (currTabId !== selectedTabId) {
+      onTabSelected(tabs.find(el => el.id === currTabId));
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={tab.id === selectedTabId ? 'is-active' : ''}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={(e) => handleClick(e.currentTarget)}
```

Comment:
I'd pass `tab.id` to `handleClick` - any way you are mapping them and this value will always be available and correct